Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -454,7 +453,7 @@ fileprivate extension _EasyHandle {
             }
             var length = Double()
             try! CFURLSession_easy_getinfo_double(handle.rawHandle, CFURLSessionInfoCONTENT_LENGTH_DOWNLOAD, &length).asError()
```

Comment:
I didn't realise that we rely on the system libcurl. Ubuntu 14.04 LTS ships 7.35 which is even older 😢